.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,99 @@
+name: 🐛 Bug Report
+description: Report an issue with the discord-bot
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## 🐛 Bug Report
+
+        Thank you for taking the time to report an issue. Please help us diagnose the problem by filling out the following information.
+
+  - type: input
+    id: title
+    attributes:
+      label: 🧵 Bug Summary
+      description: Concisely describe the issue you're experiencing.
+      placeholder: "e.g. /ping command doesn't respond"
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps
+    attributes:
+      label: 📋 Steps to Reproduce
+      description: Provide a clear set of steps to reproduce the issue.
+      placeholder: |
+        1. Start the bot using `python3 main.py`
+        2. Send `/ping` in the server
+        3. Observe that there is no response
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: ✅ Expected Behavior
+      placeholder: e.g. "Should reply with 'Pong!'"
+    validations:
+      required: true
+
+  - type: textarea
+    id: actual
+    attributes:
+      label: ❌ Actual Behavior
+      placeholder: e.g. "No response, or error in log"
+    validations:
+      required: true
+
+  - type: input
+    id: commit
+    attributes:
+      label: Commit SHA or name
+      description: Paste the commit hash or branch name you were using when the bug occurred (e.g. `abc1234`, `main@HEAD`, or `feature/xyz`).
+      placeholder: e.g. abc1234 or feature/commands-refactor
+    validations:
+      required: true
+
+  - type: input
+    id: environment
+    attributes:
+      label: 🧪 Runtime Environment
+      description: OS, Python version, Docker (if used), Discord.py version, etc.
+      placeholder: e.g. "Ubuntu 24.04, Python 3.11, discord.py 2.3.2"
+    validations:
+      required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: 📜 Logs / Traceback
+      description: Paste any relevant terminal logs, exceptions, or errors.
+      render: shell
+    validations:
+      required: true
+
+  - type: dropdown
+    id: area
+    attributes:
+      label: 📁 Area Affected
+      multiple: true
+      options:
+        - Commands (e.g. /ping, /help)
+        - Event handling
+        - Permissions
+        - Database/Storage
+        - Discord API
+        - Bot startup/shutdown
+        - CI / pre-commit
+        - Other
```

Comment:
Add another option related to bugs in the response sent (format, style, content)